### PR TITLE
add create clusterrolebinding command

### PR DESCRIPTION
**Pull request by deads2k**
Adds `kubectl create clusterrolebinding`.

@kubernetes/sig-cli

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37098)

### CODE REVIEW

**Comment by deads2k** [comment]
@fabianofranz not sure what you'd like to do downstream, but this is the correct name here.

**Comment by pwittrock** [comment]
@deads2k I am out next week.  Do you need me to review this by EOD, or can this wait a couple weeks since we are focussed on 1.5 stuff anyway?

**Comment by deads2k** [comment]
> @deads2k I am out next week. Do you need me to review this by EOD, or can this wait a couple weeks since we are focussed on 1.5 stuff anyway?

It can wait.  We could also let the bot re-assign.

**Comment by pwittrock** [comment]
SGTM

**Comment by fabianofranz** [comment]
cc @liggitt

File: pkg/kubectl/cmd/util/factory.go [hunk 1]
```diff
@@ -206,6 +206,7 @@ const (
 	SecretForDockerRegistryV1GeneratorName      = "secret-for-docker-registry/v1"
 	SecretForTLSV1GeneratorName                 = "secret-for-tls/v1"
 	ConfigMapV1GeneratorName                    = "configmap/v1"
+	ClusterRoleBindingV1GeneratorName           = "clusterrolebinding.rbac.authorization.k8s.io/v1alpha1"
```
Comment (soltysh) [inline pkg/kubectl/cmd/util/factory.go]: I'm a little bit hesitant to use this lengthy generator name. Although this one will not be probably exposed to users.

**Comment by soltysh** [comment]
One question/doubt. Overall LGTM.

File: pkg/kubectl/cmd/util/factory.go (hunk 1 above)
Comment (deads2k) [inline pkg/kubectl/cmd/util/factory.go]: > I'm a little bit hesitant to use this lengthy generator name. Although this one will not be probably exposed to users.

We get a release to change it if we don't like it.  If we expect these to take off (I still think its unlikely), we'll need a reasonable way to namespace them.

Comment (soltysh) [inline pkg/kubectl/cmd/util/factory.go]: My main concern is when using `kubectl run` we advice users to use `--generator` flag to be sure this works consistently between versions. But I guess the current directions are to get rid of run and rather replace it with targeted `kubectl create` commands. That's why I guess I'm good with that.

**Comment by deads2k** [comment]
@k8s-bot verify test this

**Comment by deads2k** [comment]
@k8s-bot cvm gke e2e test this

File: docs/user-guide/kubectl/kubectl_create_clusterrolebinding.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+This file is autogenerated, but we've stopped checking such files into the
+repository to reduce the need for rebases. Please run hack/generate-docs.sh to
+populate this file.
+
+<!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
```
Comment (lavalamp) [inline docs/user-guide/kubectl/kubectl_create_clusterrolebinding.md]: This analytics tag shouldn't be here, I'm pretty sure?

**Comment by lavalamp** [comment]
Shouldn't this have a test in test-cmd.sh or something?

**Comment by deads2k** [comment]
> Shouldn't this have a test in test-cmd.sh or something?

It does.  See https://github.com/kubernetes/kubernetes/pull/37098/files#diff-4b1bb481b0e7f5daa0fdf4675a208493R425

File: docs/user-guide/kubectl/kubectl_create_clusterrolebinding.md (hunk 2 above)
Comment (deads2k) [inline docs/user-guide/kubectl/kubectl_create_clusterrolebinding.md]: > This analytics tag shouldn't be here, I'm pretty sure?

Purely generated and all the others have this too.  See https://raw.githubusercontent.com/kubernetes/kubernetes/master/docs/user-guide/kubectl/kubectl_create_secret.md

**Pull request merged**